boltdesignsystem/bolt
# WWW-793: Closed captioning default display issues

## Jira

https://pegadigitalit.atlassian.net/browse/WWW-793

## Summary

Updated the Brightcove video examples in Patternlab

## Details

* Added a video with localized translation to Patternlab to test closed captioning
* Updated the first play event to a more stable version (refer to https://docs.brightcove.com/brightcove-player/1.x/Player.html#firstplay-event)

## To Test

* Pull down the branch and start Bolt
* Open http://localhost:3000/pattern-lab/?p=components-accordion-content-variations
* Open the "Expand to see a video" tab and play the video, you should see captions present on the video
** (Optionally you can change the lang tag on the HTML to test the other languages, "it" for example should automatically play the Italian captions)

To test the ".one("play"..." adjustment repeat the process above in Safari and confirm you see the captions
